python/ray/_private/utils.py

```diff
@@ -1899,3 +1899,20 @@ def try_import_each_module(module_names_to_import: List[str]) -> None:
             importlib.import_module(module_to_preload)
         except ImportError:
             logger.exception(f'Failed to preload the module "{module_to_preload}"')
+
+
+def update_envs(env_vars: Dict[str, str]):
```

Comment:
docstring?

Reply:
done. But not particularly well described.